bin/node/runtime/src/lib.rs

```diff
@@ -713,7 +700,6 @@ pub type SignedExtra = (
 	frame_system::CheckNonce<Runtime>,
 	frame_system::CheckWeight<Runtime>,
 	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
-	pallet_contracts::CheckBlockGasLimit<Runtime>,
```

Comment:
It brakes chains which use the contracts module as they will be referencing the now gone signed extension.

What are you saying here?

Reply:
Hm, what do you mean by "referencing"? I was thinking that this **could have** been a breaking change if `CheckBlockGasLimit` required including some data into the extrinsics extras (but AFAIK it didn't).

Reply:
Okay then you need to define "braking". Because every chain node compiled against an older version of substrate that uses the contracts pallet will no longer compile after this change. That is what I call braking change.

And yes you are right. It did only validation. No inclusion of additional data.

Reply:
By "breaking" i meant that the format of the extrinsics would have been incompatible.

Reply:
What would be the implication of that? I mean you would import the old extrinsics with a compatible runtime then upgrade the runtime and expect other extrinsics.